app.py

```diff
@@ -18,8 +18,10 @@
 # SampleOrientedIndexer as SampleIndexer, \
 # ProjectOrientedIndexer as ProjectIndexer
 from chalicelib.utils import DataExtractor
+from utils.indexer import BaseIndexer
```

Comment:
@jyotheeshwar Can you check if this works when deploying to AWS lambda? I know that when packaging the application and pushing to AWS, all the modules must be packed inside a folder called`chalicelib`. Does your automated deployment strategy deal with that? See:
http://chalice.readthedocs.io/en/latest/topics/packaging.html